src/Tribe/Ticket_Object.php

```diff
@@ -272,6 +298,43 @@ public function stock( $value = null ) {
 		}
 
 		/**
+		 * Sets of gets the current global stock mode in effect for the ticket.
+		 *
+		 * Typically this is one of the constants provided by Tribe__Tickets__Global_Stock:
+		 *
+		 *     GLOBAL_STOCK_MODE if it should draw on the global stock
+		 *     CAPPED_STOCK_MODE as above but with a limit on the total number of allowed sales
+		 *     OWN_STOCK_MODE if it should behave as if global stock is not in effect
+		 *
+		 * @param string $mode
+		 *
+		 * @return string
+		 */
+		public function global_stock_mode( $mode = null ) {
+			if ( is_string( $mode ) ) {
+				$this->global_stock_mode = $mode;
+			} elseif ( null === $mode ) {
+				return $this->global_stock_mode;
+			}
+		}
+
+		/**
+		 * Sets or gets any cap on sales that might be in effect for this ticket when global stock
+		 * mode is in effect.
+		 *
+		 * @param int $cap
+		 *
+		 * @return int
+		 */
+		public function global_stock_cap( $cap = null ) {
+			if ( is_numeric( $cap ) ) {
+				$this->global_stock_cap = (int) $cap;
+			} elseif ( null === $cap ) {
+				return (int) $this->global_stock_cap;
```

Comment:
Essentially the same question as [the one](https://github.com/moderntribe/event-tickets/pull/113/files#r52367872) on `global_stock_mode()`.